lib/modules/events.js

```diff
@@ -4,7 +4,11 @@ import { findNodeHandle, NativeModules } from 'react-native';
 const { AREventsModule } = NativeModules;
 
 function postEvent(component, info) {
-  AREventsModule.postEvent(findNodeHandle(component), info);
+  try {
+    AREventsModule.postEvent(findNodeHandle(component), info);
+  }
```

Comment:
Ideally we'd only wrap the findNodeHandle call, store the resulting reactTag, and do the rest outside the try/catch block, so we don't mask other potential problems.